benchmarks/src/jvmMain/kotlin/space/kscience/kmath/benchmarks/SVDBenchmark.kt

```diff
@@ -0,0 +1,34 @@
+/*
+ * Copyright 2018-2021 KMath contributors.
+ * Use of this source code is governed by the Apache 2.0 license that can be found in the license/LICENSE.txt file.
+ */
+
+package space.kscience.kmath.benchmarks
+import kotlinx.benchmark.Benchmark
+import kotlinx.benchmark.Blackhole
+import kotlinx.benchmark.Scope
+import kotlinx.benchmark.State
+import space.kscience.kmath.tensors.core.BroadcastDoubleTensorAlgebra.svd
+import space.kscience.kmath.tensors.core.DoubleTensorAlgebra
+import space.kscience.kmath.tensors.core.DoubleTensorAlgebra.Companion.svdPowerMethod
+
+@State(Scope.Benchmark)
+class SVDBenchmark {
+    companion object {
+        val tensor = DoubleTensorAlgebra.randomNormal(intArrayOf(10, 10, 10), 0)
```

Comment:
please run several benchmarks small - (5,5) , medium (10,10) - large (50,50) - very large (100,100). In the initialisation assert that norm(A - U S Vt) < 1e-9